polaris-react/src/components/Modal/README.md

```diff
@@ -651,7 +651,7 @@ function ModalWithoutTitleExample() {
           },
         ]}
       >
-        <Modal.Section>
+        <Modal.Section titleHidden>
```

Comment:
I wonder if this wouldn't have been better named 'modalTitleHidden'. The reason I bring it up is that this was confusing to me since Section does not have a title prop.